spec/system/drink_buttons_spec.rb

```diff
@@ -185,13 +185,27 @@
     end
 
     describe "clicking open button of sealed bottle", js: true do
+      before do
+        accept_confirm do
+          click_link "open-button-#{sealed_sake.id}"
+        end
+      end
+
+      it "redirects to index page" do
+        expect(page).to have_current_path(sakes_path)
+      end
+
+      it "has success flash message" do
+        expect(page).to have_css(".alert-success")
+      end
+
+      it "has link to the updated sake" do
```

Comment:
itが示すのは`describe "clicking open button of sealed bottle", js: true do`のクリック動作かそれによるページのことなので、以下のように書くといいと思う。
```suggestion
      it "has flash message which has link to the updated sake" do
```
もしくは、`it "makes flash message which....`とか

Reply:
thx
DeepL先生に聞いたらwhich has よりcontaining のほうがよさそうだった

Reply:
Fixed at 237c6a4